spotinst/azure_v3/stateful_node_azure_vm_sizes/fields_spotinst_stateful_node_azure_vm_sizes.go

```diff
@@ -2,164 +2,166 @@ package stateful_node_azure_vm_sizes
 
 import (
 	"fmt"
+	"github.com/spotinst/spotinst-sdk-go/service/stateful/providers/azure"
```

Comment:
please import the required sdk

Reply:
This is the sdk used for the stateful node, checked with other stateful node fields